src/scope/scope.ts

```diff
@@ -0,0 +1,50 @@
+import { BindingScopeEnum, interfaces } from "../inversify";
+import { isPromise } from "../utils/async";
+
+export const _tryGetFromScope = <T>(
```

Comment:
An export function is public so the underscore symbol should be removed

Reply:
Agreed. I was on an underscore roll.